src/main/java/com/google/devtools/build/lib/runtime/UiOptions.java

```diff
@@ -178,6 +234,17 @@ public ProgressModeConverter() {
       help = "Report all events known to the Bazel UI.")
   public boolean experimentalUiDebugAllEvents;
 
+  @Option(
+      name = "ui_event_filters",
+      converter = EventFiltersConverter.class,
+      defaultValue = "null",
+      metadataTags = {OptionMetadataTag.HIDDEN},
+      documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
+      effectTags = {OptionEffectTag.UNKNOWN},
```

Comment:
These should be `OptionDocumentationCategory.LOGGING` and `OptionEffectTag.TERMINAL_OUTPUT`.